### PR TITLE
Completed-CSS-Ecommerce-Page

### PR DESCRIPTION
**Pull request by Henry-Nichols**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude!! A couple of visual fixes needed:
1. Your images are a bit too large and seem to be overlapping each other. You may just need to put more space between them, but you might also need to adjust the size of the images.
2. The background color for the current price is too dark as compared to the example. I would use a color like "peach"

Address these items and push up your changes, and I will take a closer look at this. Make sure you attach an updated screenshot once your work has been updated!

**Comment by Henry-Nichols** [comment]
image

File: css-ecommerce-page/style.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  font-size: 16px;
+}
+
+h1 {
+  font-weight: 600;
+}
+
+span {
+  opacity: 0.3;
+}
```
Comment (codingcodymiller) [inline css-ecommerce-page/style.css:20]: Is this to make the text lighter? This is actually making your content more see-through, this is not the ideal way to do this. You should just change the text color to something lighter

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Henry-Nichols** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**